src/components/WebexActivity/WebexActivity.js

```diff
@@ -71,8 +72,4 @@ export default function WebexActivity(props) {
 
 WebexActivity.propTypes = {
   activityID: PropTypes.string.isRequired,
```

Comment:
Remove trailing comma
```suggestion
  activityID: PropTypes.string.isRequired
```

Reply:
you cant. prettify requires it.

Reply:
I see why this is off, we shouldn't be passing an adapter to `Header` anymore, it need it for the `WebexAvatar`, but `WebexAvatar` has its own adapter now.